lib/versioned/^4.0.0-alpha.2/index.js

```diff
@@ -48,13 +51,15 @@ function execute(opts, env) {
     }
     if (opts.tasks) {
       tree = gulpInst.tree({ deep: true });
-      tree.label = 'Tasks for ' + chalk.magenta(tildify(env.configPath));
+      tree.label = isString(config.description) ? config.description :
+        'Tasks for ' + chalk.magenta(tildify(env.configPath));
 
       return logTasks(tree, opts.depth, getTask(gulpInst));
     }
     if (opts.tasksJson) {
       tree = gulpInst.tree({ deep: true });
-      tree.label = 'Tasks for ' + tildify(env.configPath);
+      tree.label = isString(config.description) ? config.description :
```

Comment:
It's not enough to check `isString`.  We don't want to print an empty string description